src/main/java/me/zeroeightsix/kami/module/modules/combat/VisualRange.kt

```diff
@@ -41,6 +42,7 @@ class VisualRange : Module() {
         if (tickPlayerList.size > 0) {
             for (playerName in tickPlayerList) {
                 if (playerName == mc.player.name) continue
+                if (Friends.isFriend(playerName) && !friends.value) continue
```

Comment:
could you collapse these into the same if statement?

Reply:
so something along the lines of 
`if (playerName == mc.player.name || Friends.isFriend(playerName) && !friends.value) continue`?

Reply:
```kotlin
                if ((playerName == mc.player.name) || (Friends.isFriend(playerName) && !friends.value)) continue
```

Reply:
Ahh. Gotcha. I just made the change. The reason I went with it on two lines originally is I thought it would make more sense that way having two different checks for two different things be on different lines, but If you think it makes more sense on one line then that's how it is now if you want to merge it.